nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -688,6 +690,10 @@ impl<'a> BlueprintBuilder<'a> {
         Blueprint {
             id: blueprint_id,
             sleds,
+            pending_mgs_updates: self
```

Comment:
Similar to above - do we want an issue for adding builder / planner support?

Reply:
Yeah, there's already #7414.  I've also got a project task for reconfigurator-cli support, which will cover the builder part potentially sooner than 7414.